docs/Getting Started/Arch Linux/Arch Linux Root on ZFS.rst

```diff
@@ -0,0 +1,986 @@
+
+   For example, change::
+
+    zpool create \
+      ... \
+      ${DISK}-part2
+
+   to::
+
+    zpool create \
+      ... \
+      mirror \
+      /dev/disk/by-id/ata-disk1-part2
+      /dev/disk/by-id/ata-disk2-part2
+
+   if needed, replace ``mirror`` with ``raidz1``, ``raidz2`` or ``raidz3``.
+
+#. Create boot pool::
+
+    zpool create \
+        -o ashift=12 \
+        -d -o feature@async_destroy=enabled \
+        -o feature@bookmarks=enabled \
+        -o feature@embedded_data=enabled \
+        -o feature@empty_bpobj=enabled \
+        -o feature@enabled_txg=enabled \
+        -o feature@extensible_dataset=enabled \
+        -o feature@filesystem_limits=enabled \
+        -o feature@hole_birth=enabled \
+        -o feature@large_blocks=enabled \
+        -o feature@lz4_compress=enabled \
+        -o feature@spacemap_histogram=enabled \
+        -O acltype=posixacl \
+        -O canmount=off \
+        -O compression=lz4 \
+        -O devices=off \
+        -O normalization=formD \
+        -O relatime=on \
+        -O xattr=sa \
+        -O mountpoint=/boot \
+        -R $INST_MNT \
+        bpool_$INST_UUID \
+        ${DISK}-part2
+
+   You should not need to customize any of the options for the boot pool.
+
+   GRUB does not support all of the zpool features. See ``spa_feature_names``
+   in `grub-core/fs/zfs/zfs.c
+   <http://git.savannah.gnu.org/cgit/grub.git/tree/grub-core/fs/zfs/zfs.c#n276>`__.
+   This step creates a separate boot pool for ``/boot`` with the features
+   limited to only those that GRUB supports, allowing the root pool to use
+   any/all features. Note that GRUB opens the pool read-only, so all
+   read-only compatible features are “supported” by GRUB.
+
+   **Feature Notes:**
+
+   - The ``allocation_classes`` feature should be safe to use. However, unless
+     one is using it (i.e. a ``special`` vdev), there is no point to enabling
+     it. It is extremely unlikely that someone would use this feature for a
+     boot pool. If one cares about speeding up the boot pool, it would make
+     more sense to put the whole pool on the faster disk rather than using it
+     as a ``special`` vdev.
+   - The ``project_quota`` feature has been tested and is safe to use. This
+     feature is extremely unlikely to matter for the boot pool.
+   - The ``resilver_defer`` should be safe but the boot pool is small enough
+     that it is unlikely to be necessary.
+   - The ``spacemap_v2`` feature has been tested and is safe to use. The boot
+     pool is small, so this does not matter in practice.
+   - As a read-only compatible feature, the ``userobj_accounting`` feature
+     should be compatible in theory, but in practice, GRUB can fail with an
+     “invalid dnode type” error. This feature does not matter for ``/boot``
+     anyway.
+
+#. Create root pool:
+
+   - Unencrypted::
+
+      zpool create \
+        -o ashift=12 \
+        -O acltype=posixacl \
+        -O canmount=off \
+        -O compression=zstd \
```

Comment:
Just to be sure - archzfs already ships 2.0 with zstd? Won't there be a case with zfs 0.8 branch?

Reply:
[archzfs](https://archzfs.com/archzfs/x86_64/) always ships the latest version of zfs packages for both prebulit and DKMS packages. Currently it's 2.0.1.